modules/ebi-metagenomics/sanntis/main.nf

```diff
@@ -0,0 +1,50 @@
+process SANNTIS {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'biocontainers/sanntis:0.9.3.5--pyhdfd78af_1' :
+        'biocontainers/sanntis:0.9.3.5--pyhdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(interproscan_output), path(gbk)
+
+    output:
+    tuple val(meta), path("*_sanntis.gff"), emit: gff
+    path "versions.yml"           , emit: versions
```

Comment:
comma under comma :D

Reply:
Done in https://github.com/EBI-Metagenomics/nf-modules/commit/ad6b4bbca5144c2f7659e3d4b8c1d3e1c648f5b4